releaf.gemspec

```diff
@@ -38,5 +38,4 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'spring-commands-rspec'
   s.add_development_dependency 'roo'
 
-  s.required_ruby_version = '>= 2.2.0'
```

Comment:
There are "releaf" gem, not "releaf-core" that are used in rails projects. And there will be no single "releaf-core" gem release ever. All minimum requirements (rails, ruby versions. Also rails have ruby version defined in rails.gemspec.